src/components/votingV2/index.js

```diff
@@ -5,9 +5,18 @@ import { withRouter } from 'react-router';
 import VotingV2 from './votingV2';
 import { clearVotes, votePlaced } from '../../actions/voting';
 
+const filterObjectPropsWithValue = (object = {}, value) => (
+  Object.keys(object).filter(key => object[key] === value)
+);
```

Comment:
IMO would be better to have this function into the `utils/helpers.js` file since it could be useful in other places in the future, and not only voting related